### DISCUSSION TITLE
unable to connect tally jdbc driver in dbeaver

### DISCUSSION THREAD

**Discussion by Jaydeb91** [dbeaver/dbeaver#12497] [Q&A] [answered]
Hi All,
Can you please help to understand - Why I am unable to install JDBC driver in dbeaver

your quick response will be highly appreciated!

![Tally Issue-db](https://user-images.githubusercontent.com/52272413/112601630-aefc1180-8e38-11eb-9ba2-7553190424c6.png)

Getting JDBC driver from Cdata @ https://www.cdata.com/kb/tech/tally-jdbc-dbvr.rst

Thanks, Jaydeb

**Comment by kseniiaguzeeva** [comment]
What exactly goes wrong? Do you make all steps from the instruction successfully? Could you send a screenshot of settings tab of **Edit Driver** dialog?

**Comment by Jaydeb91** [comment]
Yes, I am following all the instruction given by the cdata @ https://www.cdata.com/kb/tech/tally-jdbc-dbvr.rst

  screenshot settings tab of **Edit Driver** dialog 
![image](https://user-images.githubusercontent.com/52272413/112621734-1bced600-8e50-11eb-9718-c8cb42108a56.png)

**Comment by kseniiaguzeeva** [comment]
Jar file looks incorrect (class doesn't appear). Most probably it is Tally issue.

**Comment by uslss** [comment, marked as answer]
@Jaydeb91 Edit driver dialog was changed in recent versions
You could add Driver at Libraries tab
![image](https://user-images.githubusercontent.com/31996417/117804381-61a8f780-b260-11eb-8553-6869e1369054.png)
